### ISSUE TITLE
Unable to start BCG+ 2.3.0 Missing necronomicon 1.0.1

### ISSUE THREAD

**Issue by Lobstram** [bigchadguys/bcgplus#37]
Hi!

I just upgraded my server from `2.2.0` to `2.3.0`, but it won't start and gives the following error:

```
mc-1  | [04:10:06] [main/ERROR]: Incompatible mods found!
mc-1  | net.fabricmc.loader.impl.FormattedException: Some of your mods are incompatible with the game or each other!
mc-1  | A potential solution has been determined, this may resolve your problem:
mc-1  | 	 - Install necronomicon, version 1.0.1 or later.
mc-1  | More details:
mc-1  | 	 - Mod 'You Shall Not Spawn!' (ysns) 1.0.5 requires version 1.0.1 or later of necronomicon, which is missing!
mc-1  | 	at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:51) ~[fabric-loader-0.16.0.jar:?]
mc-1  | 	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:196) ~[fabric-loader-0.16.0.jar:?]
mc-1  | 	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:146) ~[fabric-loader-0.16.0.jar:?]
mc-1  | 	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68) ~[fabric-loader-0.16.0.jar:?]
mc-1  | 	at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) ~[fabric-loader-0.16.0.jar:?]
mc-1  | 	at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:69) ~[fabric-loader-0.16.0.jar:?]
mc-1  | 	at net.fabricmc.installer.ServerLauncher.main(ServerLauncher.java:69) ~[fabric-server-mc.1.20.1-loader.0.16.0-launcher.1.0.1.jar:1.0.1]
mc-1  | 2024-07-13T04:10:06.531Z	WARN	mc-server-runner	Minecraft server failed. Inspect logs above for errors that indicate cause. DO NOT report this line as an error.{"exitCode": 1}
```

I am running my server via Docker Compose:

```
services:
  mc:
    image: itzg/minecraft-server:java17-jdk
    volumes:
      - ./data:/data
    environment:
      MEMORY: 8G
      EULA: "true"
      TYPE: AUTO_CURSEFORGE
      CF_SLUG: "bigchadguys-plus"
      CF_API_KEY: ${CF_API_KEY}
    ports:
      - "25565:25565"
```

**Comment by jflec** [comment]
Have you confirmed that https://www.curseforge.com/minecraft/mc-mods/necronomicon/files/5091307 is in your modlist? Server files aren't publicly available yet so something might've gone wrong during the installation phase.

**Comment by Lobstram** [comment]
> Have you confirmed that https://www.curseforge.com/minecraft/mc-mods/necronomicon/files/5091307 is in your modlist? Server files aren't publicly available yet so something might've gone wrong during the installation phase.

I'm not sure what that means within the context of the Docker image that I'm using: https://docker-minecraft-server.readthedocs.io/en/latest/types-and-platforms/mod-platforms/auto-curseforge/ I've always just let it do its thing and haven't touched a single file myself.

I tried starting a brand new server, but got the same error on startup.

I was able to resolve it by downloading `Necronomicon-Fabric-1.4.2.jar` from the link that you provided and putting it into my `mods` folder, but I'm not sure if that's the most appropriate way to fix it?
